OpenRA.Mods.RA2/Activities/EnterGarrison.cs

```diff
@@ -56,7 +56,7 @@ protected bool TryStartEnter(Actor self, Actor targetActor)
 
 			// Make sure we can still enter the transport
 			// (but not before, because this may stop the actor in the middle of nowhere)
-			if (enterGarrison == null || !garrisoner.Reserve(self, enterGarrison))
+			if (enterGarrison == null || !garrisoner.Reserve(self, enterGarrison) || enterGarrison.IsTraitDisabled || enterGarrison.IsTraitPaused)

```

Comment:
I'd shift the reserving check to last one tbh since that's more expensive than the boolean lookups.

Reply:
Done.